### PR TITLE
Peep::ItemFlags merging

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Continuing on from #13311
This will break replays. 
@Gymnasiast could you confirm if SV4's have extra item flags. For some reason we weren't importing them.

### CODE REVIEW

**Review by tupaschoal — APPROVED** [review]
Seems alright, though do wait for Gym's confirmation

**Comment by Gymnasiast** [comment]
Do I understand correctly that PeepItemExtraFlags can (only)contain the following flags:
```
   PEEP_ITEM_PHOTO2 = (1 << 0),
    PEEP_ITEM_PHOTO3 = (1 << 1),
    PEEP_ITEM_PHOTO4 = (1 << 2),
    PEEP_ITEM_PRETZEL = (1 << 3),
    PEEP_ITEM_CHOCOLATE = (1 << 4),
    PEEP_ITEM_ICED_TEA = (1 << 5),
    PEEP_ITEM_FUNNEL_CAKE = (1 << 6),
    PEEP_ITEM_SUNGLASSES = (1 << 7),
    PEEP_ITEM_BEEF_NOODLES = (1 << 8),
    PEEP_ITEM_FRIED_RICE_NOODLES = (1 << 9),
    PEEP_ITEM_WONTON_SOUP = (1 << 10),
    PEEP_ITEM_MEATBALL_SOUP = (1 << 11),
    PEEP_ITEM_FRUIT_JUICE = (1 << 12),
    PEEP_ITEM_SOYBEAN_MILK = (1 << 13),
    PEEP_ITEM_SU_JONGKWA = (1 << 14),
    PEEP_ITEM_SUB_SANDWICH = (1 << 15),
    PEEP_ITEM_COOKIE = (1 << 16),
    PEEP_ITEM_EMPTY_BOWL_RED = (1 << 17),
    PEEP_ITEM_EMPTY_DRINK_CARTON = (1 << 18),
    PEEP_ITEM_EMPTY_JUICE_CUP = (1 << 19),
    PEEP_ITEM_ROAST_SAUSAGE = (1 << 20),
    PEEP_ITEM_EMPTY_BOWL_BLUE = (1 << 21),
```
None of these were in RCT1, which leads me to believe that RCT1 did not have ItemExtraFlags.

File: src/openrct2/rct1/RCT1.h
Comment (Gymnasiast) [inline src/openrct2/rct1/RCT1.h:471]:
```suggestion
        return item_standard_flags;
```

or just set directly from `item_standard_flags`.

Comment (duncanspumpkin) [reply src/openrct2/rct1/RCT1.h:471]: cool sounds good. Strange that CS would have space for the flags and then not use them. Must have been something that he decided wasn't required in the end.

File: src/openrct2/peep/Guest.cpp
Comment (Gymnasiast) [inline src/openrct2/peep/Guest.cpp:5344]: I'm not sure I understand this bit. Why was adding `ShopItem::Photo2` needed previously?

Comment (duncanspumpkin) [reply src/openrct2/peep/Guest.cpp:5344]: It was a mistake photo 2 is the same as 32. So it was adding it twice

Comment (Gymnasiast) [reply src/openrct2/peep/Guest.cpp:5344]: OK, and do I understand correctly this adding of 32 is only temporary until these flags are merged in https://github.com/OpenRCT2/OpenRCT2/pull/13513 ?

Comment (duncanspumpkin) [reply src/openrct2/peep/Guest.cpp:5344]: Yes

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**
